davinci/src/main/kotlin/com/pingidentity/davinci/module/Connector.kt

```diff
@@ -21,6 +22,8 @@ import kotlinx.serialization.json.jsonObject
 import kotlinx.serialization.json.jsonPrimitive
 import kotlinx.serialization.json.put
 
+internal const val CONTINUE_NODE = "com.pingidentity.davinci.CONTINUE_NODE"
```

Comment:
Do we have anywhere more centralized that we hold the consts?

Reply:
There is not centralized place to store constant, will review later and see if that is necessary.